ddtrace/constants.py

```diff
@@ -36,3 +36,6 @@
 AUTO_KEEP = 1
 # Use this to explicitly inform the backend that a trace should be kept and stored.
 USER_KEEP = 2
+
+MAX_KINESIS_DATA_SIZE = 1 << 20  # 1MB
+MAX_EVENTBRIDGE_DETAIL_SIZE = 1 << 18  # 256KB
```

Comment:
These should not be in this module as these are constants specific to the core library and not integrations.

These can just live in the `botocore.patch` module